new-pipeline/src/dist.js

```diff
@@ -67,11 +67,11 @@ $(function () {
       .multiselect("select", gInitialPageState.sort_keys);
     $("input[name=table-toggle][value=" + (gInitialPageState.table !==
```

Comment:
`gInitialPageState` contains the configuration of how the user previously looked at the page. They may have changed all of the advanced settings in any number of ways.

Now that we're using checkboxes instead of radio buttons, this will have to be reworked. It should be more straightforward now as we simply need to set a checkbox to `checked` if the page state says it's non-zero, or uncheck it otherwise.

So something like

```js
$("input[name=cumulative-toggle]").checked = gInitialPageState.cumulative !== 0;
$("input[name=cumulative-toggle]").trigger("change");
```

for all of the checkboxes. I'm not sure we need to trigger a `change` on all of them, but we'd better leave it in there for now, just in case.